### PR TITLE
[Term Entry] Python Strings `.maketrans()` method

### LINKED ISSUE #3330: [Term Entry] Python Strings .maketrans()
### Reason/inspiration (optional)

We would like a new term entry on the `.maketrans()` method for Python: [Strings](https://www.codecademy.com/resources/docs/python/strings). The entry should go in a new file under `docs/content/python/concepts/strings/terms/maketrans/maketrans.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```py `code snippet.
- A “Codebyte Example” section with a ` ```codebyte/python ` snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by manviii27**
### Description

A new term entry for Python Strings `.maketrans()` method.
Close #3330 

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/strings/terms/maketrans/maketrans.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+Title: '.maketrans()'
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/maketrans/maketrans.md:2]:
```suggestion
Title: 'maketrans()'

```

File: content/python/concepts/strings/terms/maketrans/maketrans.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+Title: '.maketrans()'
+Description: 'Used to construct the transition table.'
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/maketrans/maketrans.md:3]:
```suggestion
Description: 'Returns a transition table based on the given strings.'

```

File: content/python/concepts/strings/terms/maketrans/maketrans.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.maketrans()`** method is used for creating a translation table that can be used to perform character replacements or translations in strings. This method is particularly useful when you want to replace specific characters or substrings with other characters, or when you want to remove certain characters from a string.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/maketrans/maketrans.md:16]:
```suggestion
The **`maketrans()`** method is used for creating a translation table that can be used to perform character replacements or translations in strings. This method is particularly useful when there is a need to replace specific characters or substrings with other characters, or to remove certain characters from a string.

```

File: content/python/concepts/strings/terms/maketrans/maketrans.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+```pseudo
+string.maketrans(x,y,z)
+```
+
+- `x`: A string containing characters to be replaced.
+- `y`: A string containing characters with which `x` needs to be replaced.
+- `z`: A string containing characters to be deleted.
+
+The `.maketrans()` method returns a translation table, which can then be used with the `.translate()` method to perform the actual translations or replacements in a string.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/maketrans/maketrans.md:28]:
```suggestion
The `maketrans()` method returns a translation table, which can then be used with the `translate()` method to perform the actual translations or replacements in a string.

```

File: content/python/concepts/strings/terms/maketrans/maketrans.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+## Examples
+
+This example shows the use of `.maketrans()` mathod to replace characters in a string.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/maketrans/maketrans.md:32]:
```suggestion
This example shows the use of `maketrans()` method to replace characters in a string.

```

File: content/python/concepts/strings/terms/maketrans/maketrans.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+This code will result in the following output:
+
+```shell
+h2ll4 w4rld
+```
+
+## Codebyte Example
+
+The following code is runnable and uses the `.maketrans()` method.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/maketrans/maketrans.md:50]:
```suggestion
The following code is runnable and uses the `maketrans()` method.

```

File: content/python/concepts/strings/terms/maketrans/maketrans.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+```codebyte/python
+trans_table = str.maketrans('aeiou', '12345', 'l')
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/maketrans/maketrans.md:53]: This is the same example as above. We do not include duplicates within and entry, ideally every example should be demonstrating some new functionality of the method. Please revise the example or delete it

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@manviii27 I've added my comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by manviii27** [comment]
@caupolicandiaz I've made the changes, please review.

**Review by caupolicandiaz — APPROVED** [review]
@manviii27 This is ready for a second review

**Review by SSwiniarski — APPROVED** [review]
@manviii27 @caupolicandiaz Looks good.

**Pull request merged**